### PR TITLE
Add easy tests on expressions and statements

### PR DESCRIPTION
**Pull request by nicdard**
Some easy tests about expressions and statements, mainly control flow

If anyone has time to review it would be great :) 
I would prefer from now on to merge a PR only if reviewed from another contributor

### CODE REVIEW

File: 04/nicdard.ml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ; ("for.oat", "", "5")
+  ; ("and1.oat", "", "11")
+  ; ("and2.oat", "", "11")
+  ; ("or1.oat", "", "26")
+  ; ("or2.oat", "", "26")
+  ; ("sgstring.oat", "", "0")
+  ; ("nullref.oat", "", "0")
+  ]
+
+let nicdard_tests = prefix @@ easy_tests
```
Comment (dbernhard-0x7CD) [inline 04/nicdard.ml:28]: nicdard_tests should be `Gradedtests.executed_oat_file (prefix @@ easy_tests)

Comment (nicdard) [reply 04/nicdard.ml:28]: Oh thanks!

Comment (nicdard) [reply 04/nicdard.ml]: It should be ok now

**Review by dbernhard-0x7CD — APPROVED** [review]
LGTM

**Pull request merged**